roles/ceph-mds/tasks/containerized.yml

```diff
@@ -63,7 +63,7 @@
   changed_when: false
 
 - name: wait for mds socket to exist
-  command: "{{ docker_exec_cmd }} stat /var/run/ceph/{{ cluster }}-mds.{{ ansible_hostname }}.asok"
+  command: {{ docker_exec_cmd }} "stat /var/run/ceph/{{ cluster }}-mds.{{ ansible_hostname }}.asok || stat /var/run/ceph/{{ cluster }}-mds.{{ ansible_fqdn }}.asok"
```

Comment:
use `shell` module

Reply:
I think it shouldn't be needed because we launch a single command, docker exec